src/reynir_correct/pattern.py

```diff
@@ -1108,6 +1163,290 @@ def dir_loc_læsa(self, match: SimpleTree) -> None:
             )
         )
 
+    def mood_sub_ack(self, match: SimpleTree) -> None:
+        """ Indicative mood is used instead of subjunctive 
+            in concessive subclauses """
+        vp = match.first_match("VP > so_fh")
+        so = vp.first_match("so")
+        assert so is not None
+        start, end = so.span
+        variants = list( [ f for f in so.all_variants if not "fh" in f ] )
```

Comment:
No need for list() around list comprehension. And the condition at the end should probably be `if f != "fh"`